src/platform/user/profile/vap-svc/tests/helpers/addressValidationMessages.unit.spec.js

```diff
@@ -101,37 +101,33 @@ describe('showAddressValidationModal', () => {
   it('returns true with multiple suggestions', () => {
     const suggestedAddresses = [
       {
-        address: {
-          addressLine1: '400 N 65th St',
-          addressType: 'DOMESTIC',
-          city: 'Seattle',
-          countryName: 'USA',
-          countryCodeIso3: 'USA',
-          countyCode: '53033',
-          countyName: 'King',
-          stateCode: 'WA',
-          zipCode: '98103',
-          zipCodeSuffix: '5252',
-        },
+        addressLine1: '400 N 65th St',
```

Comment:
It didn't matter before since the function under test previously didn't access the `address` object or any of the properties on it, I've fixed the shape of the objects in the `suggestedAddresses` array.